examples/server/server.cpp

```diff
@@ -2192,6 +2197,38 @@ struct server_context {
             const std::string str_test = slot.generated_text.substr(pos);
             bool send_text = true;
 
+            if (slot.n_sent_text == 0 && slot.has_next_token && !slot.params.start_strings.empty()) {
```

Comment:
```suggestion
            // handle start_strings
            if (slot.n_sent_text == 0 && slot.has_next_token && !slot.params.start_strings.empty()) {
```

Adding comments to help other maintainers understand the code at a glance